react-app/src/native/constants/navigation.js

```diff
@@ -22,7 +22,7 @@ export default {
     renderTitle: (props) => {
       return (
         <View style={{ flex: 1, backgroundColor: 'transparent', flexDirection: 'row', alignItems: 'center', justifyContent: 'center' }}>
-          <Image style={{ width: 170, height: 52, margin: 'auto' }} source={require('../../images/header-logo.png')} />
+          <Image style={{ width: 170, height: 52, margin: 'auto' }} source={require('../../images/header-logo2.png')} />
```

Comment:
Just overwrite header-logo with the new logo.. otherwise we'll be bundling an extra image we're not using